core/src/main/java/org/fao/geonet/resources/CMISConfiguration.java

```diff
@@ -508,69 +507,91 @@ public void init() {
         if (repositoryId == null) {
             if (repositoryName != null) {
                 // Try to find the repository by name.
-                for (Repository repository:factory.getRepositories(parameters)) {
-                    if (repository.getName().equalsIgnoreCase(repositoryName)) {
-                        this.repositoryId = repository.getId();
-                        break;
+                try {
+                    for (Repository repository : factory.getRepositories(parameters)) {
+                        if (repository.getName().equalsIgnoreCase(repositoryName)) {
+                            this.repositoryId = repository.getId();
+                            break;
+                        }
                     }
+                } catch (CmisRuntimeException | CmisConnectionException e) {
+                    Log.error(Geonet.RESOURCES, "CMIS Repository ID not found for repositoryName=\"" + repositoryName +
+                        "\". " + (e.getErrorContent() == null?"":"  Got following results from cmis api call:" + e.getErrorContent()), e);
```

Comment:
Please add a space after `call:` and format the `?` operator with the required spaces.